jwst/assign_wcs/nirspec.py

```diff
@@ -453,10 +464,14 @@ def get_open_msa_slits(msa_file, msa_metadata_id, slit_y_range=[-.55, .55]):
 
     Parameters
     ----------
-        msa_file : str
-            MSA configuration file name, FITS keyword MSACONFL.
-        msa_metadata_id : int
-            The MSA meta id for the science file, FITS keyword MSAMETID.
+    msa_file : str
+        MSA configuration file name, FITS keyword ``MSAMETFL``.
```

Comment:
We might want to not use the term "configuration" here, because there is an actual MSA configuration file that gets uploaded to the spacecraft for setting the shutter states, and that's a different (simpler) kind of file (just a list of open/closed shutter indexes). We should refer to the FITS file we use as the "MSA meta data file", because it contains a lot more info than just the MSA shutter configuration.

Reply:
It was originally called "configuration", need to keep up with the name changes.